TACC/Core-Portal
# task/FP-1020: Index project after migrating

## Overview: ##
Projects are indexed on save, but that trigger does not take during an `./manage.py loaddata` command, nor in the `./manage.py migrate-projects` command. This code change indexes the project after a successful migration.

## Related Jira tickets: ##

* [FP-1020](https://jira.tacc.utexas.edu/browse/FP-1020)

## Testing Steps: ##
1. Clear ES with `docker volume rm VOLUME_NAME`, and [recreate the elasticsearch indicies ](https://github.com/TACC/Core-Portal/wiki/How-to-Build-Search-Index)
2. Ensure you have a value set for `_PORTAL_DATA_DEPOT_PROJECTS_SYSTEM_PREFIX`
3. Ensure you have at least one project already locally
4. Run `./manage.py migrate-projects`
5. Confirm the projects show up in projects search

## UI Photos:
<img width="1347" alt="Screen Shot 2021-06-09 at 5 33 32 PM" src="https://user-images.githubusercontent.com/20326896/121438035-ea42b280-c948-11eb-999f-a3f0719fec3b.png">

## Notes:
Does not work with CEPv2 systems if there are multiple users on a project, so you may need to remove one via tapis-cli.
